readme.md

```diff
@@ -10,7 +10,7 @@
 - 高级请求语法
 - 自定义违禁词表
 - 发送一段时间后自动撤回
-- 连接到私服 · NAIFU · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
+- 连接到私服 · [NAIFU](./docs/deploy/naifu.md) · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
```

Comment:
I don't think this should be a relative link. I would prefer <https://bot.novelai.dev/deploy/naifu.md> or <https://rentry.org/sdg_FAQ#naifu-novelai-model-backend-frontend>.

Reply:
Done!